server/src/modules/users/users.service.ts

```diff
@@ -65,8 +65,7 @@ export class UsersService {
   }
 
   async createNewUser(user: CreateUserDto): Promise<UserDocument> {
-    const createdUser = new this.userModel(user);
-    return createdUser.save();
+    return await this.userModel.create(user);
```

Comment:
Please correct for all services for consistency
I think _await_ is unnecessary in methods:

```
async sthMethod(){
    return sthAsync
}
```

Reply:
I will change it when adding tests to each of the services. It doesn't hurt us now.

Redundant awaits removed.